Staffbase/trigger.io-parse
# COD-888: Update Parse-SDK-Android

#### Corresponding Ticket / Related Links

- [COD-888](https://mitarbeiterapp.atlassian.net/browse/COD-888)


### What has been done?

- Update Parse-SDK-Android to version 1.20.0


### Why was this done?

- The previous version of our trigger parse fork used Parse-SDK 1.18, which internally still uses the [deprecated jobdispatcher](https://github.com/parse-community/Parse-SDK-Android/blob/3d439ffb52521c8f1f3893b40a7bbedf9adb343f/gcm/build.gradle#L36).


### How to test the changes?

See testing instructions in [this frontend PR](https://github.com/Staffbase/frontend/pull/4330).
